### DISCUSSION TITLE
Imbalanced ratio of number between health and disease !!

### DISCUSSION THREAD

**Discussion by Abouelhana** [SchlossLab/mikropml#318] [Q&A] [answered]
Is it feasible to use mikropml package if I have Imbalanced ratio of number between health (90%) and disease (10%) or it better to use the biomarker for example IgE level (outcome) as continuous variable to find bacterial genera that associated with outcome ?

**Comment by kelly-sovacool** [comment, marked as answer]
- Yes, you can use mikropml with imbalanced data. It's advised to evaluate model performance with AUPRC. There are a number of books and papers on handling imbalanced datasets, here's a place to start: https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0271260
- The outcome to classify/predict depends on the question you're trying to answer or problem you're trying to solve. You should discuss that with your colleagues as you're planning your study.
